src/azure-cli/azure/cli/command_modules/synapse/generated/_params.py

```diff
@@ -0,0 +1,956 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_three_state_flag,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import (
+    get_default_location_from_resource_group,
+    validate_file_or_dict
+)
+from ..action import (
+    AddAutoScale,
+    AddAutoPause,
+    AddLibraryRequirements,
+    AddSku,
+    AddRecurringScans,
+    AddBaselineResults,
+    AddDefaultDataLakeStorage,
+    AddConnectivityEndpoints,
+    AddPrivateEndpointConnections
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('synapse big-data-pool list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('workspace_name', type=str, help='The name of the workspace')
+
+    with self.argument_context('synapse big-data-pool show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('workspace_name', type=str, help='The name of the workspace', id_part='name')
+        c.argument('big_data_pool_name', options_list=['--name', '-n', '--big-data-pool-name'], type=str, help='Big '
```

Comment:
I like this change. Is it a general rule now to have the full name option besides `--name/-n`?

Reply:
yes.